requirements.txt

```diff
@@ -1,2 +1 @@
-Django>=1.8
```

Comment:
So as far as I knew, the only reason to include it in the requirements there was for testing, but now with `tox` testing you don't want to install `Django>=1.8` you want each version of Django tested against (currently 1.8, 1.10, 1.11). `tox` will automatically download the necessary Django versions to test against, so we no longer need it here as it would just be installing an extra dependency that isn't used.

I do understand that to USE `auditlog` you need Django, but that is already covered by the [install_requires](https://github.com/jjkester/django-auditlog/blob/master/setup.py#L13) in the `setup.py` for people installing the package.